modules/_labs/pagination/react/lib/GoTo.tsx

```diff
@@ -0,0 +1,72 @@
+import styled from '@emotion/styled';
+import {type} from '@workday/canvas-kit-react-core';
+import TextInput from '@workday/canvas-kit-react-text-input';
+import React from 'react';
+import uuid from 'uuid/v4';
+
+interface GoToProps {
+  onSubmit: (page: number) => void;
+  max: number;
+  goToLabel?: string;
+}
+const GoToLabel = styled('label')({
+  margin: 'auto 0',
+  paddingRight: '8px',
+  ...type.body,
+  ...type.variant.label,
+});
+
+const GoToWrapper = styled('div')({
+  display: 'flex',
+  paddingLeft: '12px',
+});
+
+const InputWrapper = styled('form')({
```

Comment:
This probably won't be part of another form, right?

Reply:
I say we should just ship with this as a form in Labs and see what folks say about it (since this can still be used without a "Go To" form)